src/timearray.jl

```diff
@@ -80,6 +84,12 @@ TimeArray(ta::TimeArray;
           colnames = _colnames(ta), meta = _meta(ta), args...) =
     TimeArray(timestamp, values, colnames, meta; args...)
 
+function TimeArray(data::NamedTuple; timestamp=:time, meta=nothing, args...)
```

Comment:
I think we do not need default value for `timestamp`.

Reply:
`timestamp` here is the name of the field of `NamedTuple` that store datetime... we must define a default value

Reply:
yeah, that field indicate the time index.
I think a required keyword argument is better, not optional, then user need to set it explicitly.